docs/requirements.md

```diff
@@ -83,3 +85,21 @@ If the kernel version is U.U, it means we do not have this information at the
 moment.
 
 [1]: https://github.com/iovisor/bpftrace/blob/master/INSTALL.md#linux-kernel-requirements
+
+## Container Runtime Requirements
```

Comment:
Since there is no section about containerd, readers might think containerd is not supported at all.

What about adding the following summary table:

| Orchestrator    | Container manager | Container runtime | Support                                                                           |
|-----------------|-------------------|-------------------|-----------------------------------------------------------------------------------|
| docker          | containerd        | runc              | ✔️                                                                                 |
| nerdctl         | containerd        | runc              | ✔️                                                                                 |
| Kubernetes      | containerd        | runc              | ✔️                                                                                 |
| Kubernetes      | containerd        | wasm              | ❌ (see [#1899](https://github.com/inspektor-gadget/inspektor-gadget/issues/1899)) |
| Kubernetes      | containerd        | katacontainers    | ❌                                                                                 |
| Kubernetes      | CRI-O             | runc / crun       | Kubernetes v1.20+ (see [below](#CRI-O))                                                     |
| podman as root  | podman            | runc / crun       | ✔️                                                                                 |
| podman rootless | podman            | runc / crun       | Only with Podman API enabled (see [below](#Podman-rootless))                                          |                                  |

And keep the subsections below for more details.

We can also add another table with known issues:

| Kubernetes platforms                                   | Support                                                                           |
|--------------------------------------------------------|-----------------------------------------------------------------------------------|
| Minikube                                               | ✔️                                                                                 |
| AKS, EKS, GKS                                          | ✔️                                                                                 |
| AWS Fargate, Azure Containers instances, GKE Autopilot | ❌ (see [#1320](https://github.com/inspektor-gadget/inspektor-gadget/issues/1320)) |
| OpenShift                                              | ✔️                                                                                 |
| Talos                                                  | ❌ (see [#1934](https://github.com/inspektor-gadget/inspektor-gadget/issues/1934)) |

Reply:
Done. This looks great, thanks!